sections/futures/TradeCrossMargin/EditCrossMarginLeverageModal.tsx

```diff
@@ -49,9 +48,8 @@ type DepositMarginModalProps = {
 
 export default function EditLeverageModal({ onDismiss, editMode }: DepositMarginModalProps) {
```

Comment:
Having trouble with this component when I don't have a position open.

With no position, I click "edit" to open the modal. When I click confirm, it throws an error using the notifier. I can't click on screen to dismiss the modal, and clicking the X button doesn't dismiss the modal.

Reply:
Specifically when opening this way:
![image](https://user-images.githubusercontent.com/10401554/213315701-f8f60055-1d3c-4a4a-bc63-fdcd58d0d4cb.png)

When opened from an open position it works as expected.